docs/dapp/opl/hyperlane/README.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Hyperlane Bridge
```

Comment:
I think Hyperlane is just a protocol for message passing right? Bridge is usually associated with some ERC20 tokens or other more application-level services.

```suggestion
description: Hyperlane Protocol
```